examples/getting-started-movielens/03a-Training-with-TF.ipynb

```diff
@@ -566,15 +567,11 @@
-      "text/plain": [
-       "<IPython.core.display.Image object>"
-      ]
-     },
-     "execution_count": 18,
-     "metadata": {},
-     "output_type": "execute_result"
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "('Failed to import pydot. You must `pip install pydot` and install graphviz (https://graphviz.gitlab.io/download/), ', 'for `pydotprint` to work.')\n"
+     ]
```

Comment:
Should we re-render with pydot installed?

Reply:
@benfred thanks will do asap.